src/main/java/de/adesso/anki/battle/renderers/MqqtRenderer.java

```diff
@@ -41,9 +41,12 @@ public void render(World world) {
 	        mqtt.publish("ext/bodies", bodies);
     	}
     	catch(JsonMappingException e) {
+    		e.printStackTrace();
     	}
     	catch(MqttException e) {
+    		e.printStackTrace();
     	} catch (JsonProcessingException e) {
+    		e.printStackTrace();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a logger to log this exception. ([squid:S1148](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3AS1148))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=33&issues=AWSsxY7_sk71k7lwEFI_&open=AWSsxY7_sk71k7lwEFI_)

[ISSUE_KEY]: AWSsxY7_sk71k7lwEFI_